client/src/commonMain/kotlin/com/algolia/search/endpoint/EndpointRecommend.kt

```diff
@@ -0,0 +1,34 @@
+package com.algolia.search.endpoint
+
+import com.algolia.search.model.recommend.FrequentlyBoughtTogetherQuery
+import com.algolia.search.model.recommend.RecommendationsQuery
+import com.algolia.search.model.recommend.RelatedProductsQuery
+import com.algolia.search.model.response.ResponseSearch
+import com.algolia.search.transport.RequestOptions
+
+public interface EndpointRecommend {
+
+    /**
+     * Returns recommendations for a specific model and objectID.
+     */
+    public suspend fun getRecommendations(
```

Comment:
In general, there is no need to declare `public` indeed. But, since this is a public library, we are enabling [explicit API mode](https://kotlinlang.org/docs/whatsnew14.html#explicit-api-mode-for-library-authors), with requires the following:
> Visibility modifiers are required for declarations if the default visibility exposes them to the public API. This helps ensure that no declarations are exposed to the public API unintentionally.

Reply:
👍